tests/Unit/Dummy/Customer.php

```diff
@@ -9,4 +9,5 @@ class Customer
     public int $id;
     public string $firstName;
     public string $lastName;
+    public string $birthDay;
```

Comment:
We do not need this property here, I just wanted to test that we would not have an exception if the result of Airtable contains fields that does not exist in the DTO class :x

Reply:
Yes but this field is use in the request sent to Airtable : 
````php
        // Setup the dummy HttpClient
        $httpClient = $this->createHttpClientMock(
            "https://api.airtable.com/v0/MOCK_ID/MOCK_TABLE",
            [
                "records" => [
                    [
                        "id" => "MOCK_ID",
                        "fields" => [
                            "firstName" => "MOCK_FIRST_NAME",
                            "lastName" => "MOCK_LAST_NAME",
                            "id" => 1,
                            "birthDay" => "1986-10-30"
                        ],
                        "createdTime" => '2021-05-20T20:05:01.000Z'
                    ]
                ]
            ]
        );
```

Reply:
It was a test to see if the ObjectNormalizer would throw when it finds a birthDay field that does not exist in the DTO class :) You could delete the birthDay field in the test right here more than add a birthDay property inside Customer::class ^^

Or just leave everything as it is :D

Reply:
We can let  `brithday`, just a another data like `firstName`.

Reply:
Really not that important anyway lets resolve this and go on :)